packages/app/package.json

```diff
@@ -20,8 +20,9 @@
   },
   "dependencies": {},
   "devDependencies": {
-    "@cypress-design/vue-icon": "0.20.0",
-    "@cypress-design/vue-statusicon": "0.3.0",
+    "@cypress-design/vue-button": "0.9.2",
+    "@cypress-design/vue-icon": "0.22.1",
+    "@cypress-design/vue-statusicon": "0.4.3",
```

Comment:
Bumped those as part of troubleshooting the issues with `@cypress-design/vue-button`, and now that we're pulling in the latest `@cypress-design/css` I figured we'd want to keep the updates in case there were cross-package compatibility issues